legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/sqlQueryToString/tests/testRS.pure

```diff
@@ -0,0 +1,442 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+###Pure
+Class meta::relational::redshift::tests::domain::Person
+{
+   firstName: String[1];
+   lastName: String[1];
+   age:Integer[1];
+   key:Integer[1];
+}
+
+Class meta::relational::redshift::tests::domain::Thing
+{
+   name: String[1];
+   id: Integer[1];
+}
+
+###Relational
+Database meta::relational::redshift::tests::db::rsDb
+(
+   Table "person table" (ID INT PRIMARY KEY, firstName VARCHAR(200), lastName VARCHAR(200), age INT, key INT)
+   Schema dataset1
+   (
+      Table "thing table" (ID INT PRIMARY KEY, NAME VARCHAR(200))
+   )
+
+   Join Person_Person("person table".key = {target}.key)
+)
+
+###Mapping
+Mapping meta::relational::redshift::tests::mapping::simpleRelationalMappingPersonForBQ
+(
+   meta::relational::redshift::tests::domain::Person : Relational
+   {
+      scope([meta::relational::redshift::tests::db::rsDb])
+      (
+         firstName : "person table".firstName,
+         lastName: "person table".lastName,
+         age : @Person_Person|"person table".key
+      )
+   }
+
+   meta::relational::redshift::tests::domain::Thing : Relational
+   {
+    ~primaryKey
+    (
+      [meta::relational::redshift::tests::db::rsDb]dataset1."thing table".ID
+    )
+    ~mainTable [meta::relational::redshift::tests::db::rsDb]dataset1."thing table"
+    id: [meta::relational::redshift::tests::db::rsDb]dataset1."thing table".ID,
+    name: [meta::relational::redshift::tests::db::rsDb]dataset1."thing table".NAME
+  }
+
+)
+
+###Pure
+import meta::relational::runtime::*;
+import meta::relational::metamodel::execute::*;
+import meta::relational::mapping::*;
+import meta::relational::functions::asserts::*;
+import meta::relational::runtime::*;
+import meta::relational::tests::model::simple::*;
+import meta::relational::tests::*;
+import meta::relational::functions::sqlstring::*;
+import meta::relational::tests::mapping::sqlFunction::model::domain::*;
+import meta::relational::tests::mapping::sqlFunction::model::mapping::*;
+import meta::pure::runtime::*;
+import meta::pure::profiles::*;
+
+function meta::relational::redshift::tests::testRedShiftSuite():Boolean[1]
```

Comment:
should this have before package tag for tests to work?

Reply:
actually I took this structure from function meta::json::schema::tests::testSuite():Boolean[1] within the testtojsonschema.pure file since they had the suite structure formatted like this so I thought a similar method could be used, and was using this as a function to call all tests in IDE. But you're right in that it isn't necessary, I can remove it?

Reply:
Yes I believe we can remove this as this wrapper function won't be executed by test suite.